catcher/catcher.go

```diff
@@ -82,7 +81,7 @@ func (service *Service) LastRequestBody() ([]byte, error) {
 	}
 
 	defer request.Body.Close()
-	body, err := ioutil.ReadAll(request.Body)
+	body, err := io.ReadAll(request.Body)
```

Comment:
Note `ioutil` is deprecated and internally just calls the `io` package, so I've updated all usages of `ioutil`.